packages/scripts/src/commands/e2e-data/mappings.ts

```diff
@@ -0,0 +1,40 @@
+export function click(input = "") {
```

Comment:
This mapping is very useful to have, although it may feel a little verbose for now (e.g. pathname just calls pathname) there's a good chance we will need to add more abstraction to the authoring layer so this gives us a place to manage